diem/diem
# [docker-compose] add monitoring docker compose

## Motivation

Create docker-compose for running a monitoring stack, this way people can easily add into their fullnode deployment and provide some dashboards on the stats.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/diem/diem/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

- link all the files into the public fullnode directory `ln -s ~/diem/docker/compose/monitoring/* .`
- `docker-compose -f docker-compose.yaml -f docker-compose.mon.yaml up -d`
- make sure all components are running
- load localhost:3000
![Screen Shot 2021-05-26 at 4 10 52 PM](https://user-images.githubusercontent.com/55457892/119742608-0882bb80-be3d-11eb-80d3-63560b2ff5cd.png)
